migrations/versions/rev-2021-01-07-05:19:49-3b29ea38f0cb_.py

```diff
@@ -18,11 +18,11 @@
 
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
-    op.add_column('speaker', sa.Column('rank', sa.Integer(), nullable=False, server_default='0'))
+    op.add_column('speaker', sa.Column('order', sa.Integer(), nullable=False, server_default='0'))
```

Comment:
Why adding new column, you have to rename the rank column to order

Reply:
So i need to run the migrations again?

Reply:
Changing migrations don't work. Once a migration is committed, it is run and is permanently immutable, you have to create a new migration

Reply:
Okay and position is already a column, shall I change back to order?

Reply:
Ahh, ok. Yes